CMakeLists.txt

```diff
@@ -302,29 +301,7 @@ if(NOT IOS AND NOT ANDROID)
     set(CMAKE_EXE_LINKER_FLAGS_COVERAGE "${CMAKE_EXE_LINKER_FLAGS_DEBUG} --coverage")
     set(CMAKE_SHARED_LINKER_FLAGS_COVERAGE "${CMAKE_SHARED_LINKER_FLAGS_DEBUG} --coverage")
 
-    find_package(GRPC)
-    if(GRPC_FOUND)
-
-        # If protobuf is installed to e.g. /opt/protobuf, you need to specifiy this via environment variable:
-        # `export PROTOBUF_DIR=/opt/protobuf`
-        #
-        if (DEFINED ENV{PROTOBUF_DIR})
-            message(STATUS "PROTOBUF_DIR manually set, using protobuf from $ENV{PROTOBUF_DIR}")
-            set(Protobuf_INCLUDE_DIRS "$ENV{PROTOBUF_DIR}/include")
-            set(Protobuf_PROTOC_EXECUTABLE "$ENV{PROTOBUF_DIR}/bin/protoc")
-            set(Protobuf_PROTOC_LIBRARY_DIR "$ENV{PROTOBUF_DIR}/lib")
-            set(Protobuf_PROTOC_LIBRARIES "$ENV{PROTOBUF_DIR}/lib/libprotoc.so")
-            set(Protobuf_LIBRARIES "$ENV{PROTOBUF_DIR}/lib/libprotobuf.so")
-        else()
-            message(STATUS "PROTOBUF_DIR not set, checking automatically for Protobuf")
-            find_package(Protobuf REQUIRED)
-        endif()
-
-        add_subdirectory(grpc/server)
-        add_subdirectory(grpc/python_client)
-    else()
-        message(STATUS "gRPC not found, skipping gRPC build")
-    endif()
+    add_subdirectory(grpc/server)
```

Comment:
Should we add a cmake option, so you need to enable or can disable the gRPC build?

Reply:
Checking this.

Reply:
I agree with @julianoes on this.

Reply:
I added a BUILD_DRONECORESERVER option (see last commit).